src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURES_COUNT = 5;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Drawable getRandomFigure() {
+        return switch (random.nextInt(FIGURES_COUNT)) {
+            case 0 -> new Circle(10, colorSupplier.getRandomColor());
```

Comment:
```suggestion
            case 0 -> new Circle(10, randomColor);
```

10 - magic number, use random for generating random side. For this purpose, it's better to create separate method, that will return random value and use it for each time of figure. 
Don't forget, that private methods should be placed after all public methods